src/components/chip.tsx

```diff
@@ -65,23 +70,24 @@ export const Chip: FC<ChipProps & HTMLAttributes<HTMLButtonElement>> = ({
   );
 
   return (
-    <button
-      type="button"
+    <span
+      role="button"
       className={cn(
         'chip',
         { 'chip--disabled': disabled, 'chip--compact': compact },
         className
       )}
-      title={title}
+      onKeyPress={onKeyPress}
       onClick={onClick}
-      disabled={typeof disabled === 'boolean' ? disabled : !onClick}
+      title={title}
```

Comment:
This line could probably be removed altogether, the title prop will be passed down through the destructured `props` variable a bit below